provider/metadata_test.go

```diff
@@ -66,6 +67,7 @@ func TestMetadata(t *testing.T) {
 					"item.0.key":       "foo",
 					"item.0.value":     "bar",
 					"item.0.sensitive": "false",
+					"item.0.order":     "4",
```

Comment:
BTW this is not enough for metadata items. See the comment on top of this [function](https://github.com/coder/terraform-provider-coder/blame/main/provider/provider.go#L102-L106).

PS. VS code debugger is very handy when dealing with Terraform schema.